core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -255,21 +263,21 @@ public void onFinish() {
   private ItemTouchHelper.Callback tabCallback = new ItemTouchHelper.Callback() {
     @Override
     public int getMovementFlags(@NonNull RecyclerView recyclerView,
-      @NonNull RecyclerView.ViewHolder viewHolder) {
+                                @NonNull RecyclerView.ViewHolder viewHolder) {
```

Comment:
I think these extra indentation changes can be removed too. They aren't needed and are not related to the issue that is being fixed.

Reply:
Agreed, will undo!